### ISSUE TITLE
General: v0.33.0 pip install fails

### ISSUE THREAD

**Issue by Avantol13** [GoogleCloudPlatform/google-cloud-python#5353]
In a fresh Python v2.7.12 virtualenv on linux:
```
pip install google-cloud
```

Results in:
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-3_n60m/google-cloud/setup.py", line 22, in <module>
        with open(os.path.join(PACKAGE_ROOT, 'setup-README.rst')) as file_obj:
    IOError: [Errno 2] No such file or directory: '/tmp/pip-install-3_n60m/google-cloud/setup-README.rst'
```

Note:
```
pip install google-cloud==0.32.0
```
works fine.

I believe it has to do with recent changes: https://github.com/GoogleCloudPlatform/google-cloud-python/commit/71e5d4bf94745580834b86c3e92ac4186c3115c0

**Comment by bartaelterman** [comment]
We have the same on Python 3.6

**Comment by joergrech** [comment]
I got this problem on Amazon EMR - it looks like the archive was changed yesterday at 10:30 am and something broke (maybe just the file "setup-README.rst" is missing?):
```
...
Collecting google-cloud>=0.32.0 (from mrjob)
  Downloading https://files.pythonhosted.org/packages/49/9a/943f0e217990c37e119101bbb739d86201ec1fdf59152ecc2dba93c5a8ad/google-cloud-0.33.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/mnt/tmp/pip-build-P0PK73/google-cloud/setup.py", line 22, in <module>
        with open(os.path.join(PACKAGE_ROOT, 'setup-README.rst')) as file_obj:
    IOError: [Errno 2] No such file or directory: '/mnt/tmp/pip-build-P0PK73/google-cloud/setup-README.rst'
    
    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /mnt/tmp/pip-build-P0PK73/google-cloud

(from lines 14-43 of s3://tw-emr-synonyms/tmp/logs/j-G6AEP8ORKNR2/node/i-016376a9dc3b31913/bootstrap-actions/1/stderr.gz)
```

If this is already patched: can you tell how long it will take to be distributed to the repository in "files.pythonhosted.org" and picked up by Amazon EMR?

**Comment by tseaver** [comment]
The `google-cloud` umbrella package is deprecated, and will (after 2018-06-18) no longer actually have any of the API libraries as dependencies.  Please just install the API library / libraries you actually need, e.g.:

```bash
$ /path/to/venv/bin/pip install google-cloud-datastore
```

**Comment by tseaver** [comment]
@crwilcox We likely *do* need to get the `MANIFEST.in` fixed for the umbrella package, so that it picks up the `legacy/*` files needed.

**Comment by eiva** [comment]
Python 3.4, 3.5 also failing.
mrjsob also not working. Thanks Google!

**Comment by crwilcox** [comment]
@tseaver you are correct. I will patch this today. This shouldn't be failing.

**Comment by crwilcox** [comment]
Thanks to @Avantol13  for reporting this. And thank you to the others for including additional information. The issue was that a file was not included in the manifest (`setup-README.rst`) and this caused the package to fail to install.

A patch version is up that resolves this install issue now as 0.33.1 -> https://pypi.org/project/google-cloud/

Also, note that in around a month this package will not install packages. Instead, you should install the `google-cloud-*` package(s) you require.

**Comment by Avantol13** [comment]
Thanks for the quick fix @crwilcox
